processing/src/test/java/org/apache/druid/query/timeseries/TimeseriesQueryQueryToolChestTest.java

```diff
@@ -364,7 +365,7 @@ public void testResultLevelCacheKeyWithGrandTotal()
   }
 
   @Test
-  public void testResultArraySignature()
+  public void testResultArraySignatureWithoutFloorTime()
```

Comment:
"testResultArraySignatureWithoutTimestampResultField" would be a better name.

Reply:
Done